### PR TITLE
Fixed text and added tree grid.

### PR DESCRIPTION
**Pull request by owenschoppe**
Changes proposed in this PR:
- Added tree grid description
- Clarified one description
- Removed a line that conflicted with the actual design.

### Reviewer, please refer to this "definition of done" checklist:

- [ ] Edit and review the new Tree Grid description.
- [ ] Add image from Thanh to the Tree Grid section @tquach12 

Please confirm the following before accepting this PR:
- [ ] Any acceptance criteria have been met
- [x] Functionality has been manually tested in a testing app
- [x] Any necessary unit, regression and integration tests have been added or updated
- [x] All automated tests pass including accessibility
- [x] No linting or build errors
- [x] No high priority bugs are open
- [x] All documentation has been updated
- [ ] Release notes have been updated

*After accepting, please remind developer to merge code into all appropriate development branches.*

### CODE REVIEW

File: site/design/displaying-data/index.jsx [hunk 1]
```diff
@@ -41,6 +41,17 @@ export default (
         </figure>
       </div>
 
+      {/* Tree Grid */}
+      <div className={pf('site-side-by-side grid wrap p-vertical--x-large')}>
```
Comment (3den) [inline site/design/displaying-data/index.jsx]: question: Is it currently possible to disable the prefix through a sass variable?

Comment (owenschoppe) [inline site/design/displaying-data/index.jsx]: @brandonferrua @jina Recommendations?

Comment (brandonferrua) [inline site/design/displaying-data/index.jsx]: yes, in `_config.scss` but not recommended

Comment (3den) [inline site/design/displaying-data/index.jsx]: :+1:

Comment (kaelig) [inline site/design/displaying-data/index.jsx]: It is possible but we are considering to remove this feature from our code base and might recommend using PostCSS instead.

Comment (3den) [inline site/design/displaying-data/index.jsx]: @kaelig cool, so the idea is to move from sass to postcss?

Comment (jina) [inline site/design/displaying-data/index.jsx]: No, not really. Using postcss doesn't mean you can't still use Sass. We will still use sass for our framework. :) he was just referring to the prefixing.

**Pull request closed without merging**